### ISSUE TITLE
[Broken Website] arxiv.org

### ISSUE THREAD

**Issue by sharmaeklavya2** [darkreader/darkreader#9005]
# Broken Website Report

## Website Address
https://arxiv.org and https://arxiv.org/abs/2106.13951

## Steps To Reproduce
Simply visit the website.

## Problematic Observation
Almost everything is still in light mode. Text boxes and dropdowns turn to dark mode, though.

## Screenshots
N/A

## System Information

- OS: macOS Monterey (12.3.1) on Macbook Air 2018.
- Browser: Google Chrome Version 102.0.5005.61 (Official Build) (x86_64).
- Dark Reader Version: Version 5 Preview (4.9.51).

**Comment by Gusted** [comment]
Hi @sharmaeklavya2!

I'm not able to reproduce this, does this happen on a new Chrome profile?

**Comment by sharmaeklavya2** [comment]
Yes. I tried it in the guest profile and also tried it in incognito mode.

**Comment by Gusted** [comment]
That's weird, do you have any other extensions? Or any different settings than a normal installation?

**Comment by sharmaeklavya2** [comment]
I have all other extensions disabled. I used the 'reset settings' button to reset settings, but arxiv is still not showing up properly in dark mode.

**Comment by sharmaeklavya2** [comment]
I tried it on Firefox now (version 101.0 (64-bit)). Everything works as expected. Perhaps it's a google-chrome specific issue.

**Comment by sharmaeklavya2** [comment]
I tried it on Chromium (Version 90.0.4427.0 (Developer Build) (x86_64)). I'm getting the same problem: arxiv.org is mostly still in light mode, but the other websites are working well.

(I never use Chromium btw. It got installed by some npm package, probably `puppeteer`.)

**Comment by sharmaeklavya2** [comment]
[arxiv.org's CSS](https://static.arxiv.org/static/browse/0.3.4/css/arXiv.css?v=20200727) is specifically setting background-color to white and text color to black. I used element inspector to check which CSS rules were being applied. I found that dark reader added
```css
html, body {
    background-color: #181a1b;
}
html, body {
    border-color: #736b5e;
    color: #e8e6e3;
}
```
but the browser was prioritizing arxiv.org's CSS rules. I consulted [CSS Spcificity rules](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), and observed that the `<body>` tag at arxiv.org has `id="front"`. So I added these rules to the dark reader dev tools window:
```css
#front {
    background-color: #181a1b;
    color: #e8e6e3;
}
```
and then dark mode kind-of worked (not perfectly, since there's more to dark mode than just `background-color` and `color`).

**Comment by Gusted** [comment]
> but the browser was prioritizing arxiv.org's CSS rules.

That's a problem, because as far as I can reproduce this shouldn't be happening:
![image](https://user-images.githubusercontent.com/25481501/172023795-1c032b29-af4e-4aca-86d7-08736ee2d784.png)

**Comment by sharmaeklavya2** [comment]
I see. This looks like a Chromium bug and not a Dark Reader bug. Do you have an easy workaround in mind? I can try fixing this if it's not too hard, and send a pull request.

**Comment by Gusted** [comment]
> Do you have an easy workaround in mind? I can try fixing this if it's not too hard, and send a pull request.

Not really, if chromium is prioritizing arxiv.org's CSS rule than it should be a wider issue.
